src/app/tests/suites/certification/Test_TC_BI_1_1.yaml

```diff
@@ -54,21 +54,12 @@ tests:
 
     #issue #11053 disabled steps below Global attributes missing from YAML framework
     - label: "Read the global attribute: AttributeList"
-      disabled: true
       command: "readAttribute"
       attribute: "AttributeList"
       response:
           constraints:
               type: list
 
-    - label: "Read the global attribute: CommandList"
```

Comment:
Why is this being removed?

These comments seem to apply across the board.

Reply:
They removed it from spec

Reply:
It got renamed to ClientGeneratedCommandList/ServerGeneratedCommandList, no?

Reply:
i added both ClientGeneratedCommandList/ServerGeneratedCommandList to scripts